test-e2e/ajs.test.ts

```diff
@@ -0,0 +1,28 @@
+Feature('AJS Bundle');
+
+const assert = require('assert');
+const testSite = 'https://www.library-test-site.com';
+const testWriteKey = 'TEWEu8XrcMVejk8GOulbEx7rHGyuuijV';
```

Comment:
note for our future selves:
It'd be cool to have another `Feature` where we could parameterize `testSite` and `testWriteKey` and compare the `.har` generated requests/cookies/local storage against customers websites